web/components/MunicipalityDropdown.tsx

```diff
@@ -67,7 +67,9 @@ export const MunicipalityDropdown = (props: Props): ReactElement => {
             ...params.inputProps,
           }}
           value={searchText}
-          onChange={(event: ChangeEvent<HTMLInputElement>) => setSearchText(event.target.value)}
+          onChange={(event: ChangeEvent<HTMLInputElement>) => {
+            return setSearchText(event.target.value);
+          }}
```

Comment:
why this change? like, it's not a problem or anything, but do you want to transition towards this style?